src/client/pages/PathCatalog/PathCatalog.jsx

```diff
@@ -11,27 +11,12 @@ const onTopicChange = (e, dispatch) => {
   dispatch(setCatalogTopic(e.target.value));
 };
 
-const PathCard = ({ path, pathId }) => (
```

Comment:
Again, 8 classes for this one too, replicated with the 3 above +1 for the footer